sdk/lib/core/uri.dart

```diff
@@ -582,23 +680,24 @@ abstract class Uri {
   ///
   /// Example:
   /// ```dart
-  /// Uri uri1 = Uri.parse("a://b@c:4/d/e?f#g");
-  /// Uri uri2 = uri1.replace(scheme: "A", path: "D/E/E", fragment: "G");
-  /// print(uri2);  // prints "a://b@c:4/D/E/E?f#G"
+  /// final uri1 = Uri.parse('a://b@c:4/d/e?f#g');
+  /// final uri2 =
+  ///     uri1.replace(scheme: 'C', path: 'D/E/E', fragment: 'G');
+  /// print(uri2); // c://b@c:4/D/E/E?f#G
   /// ```
   /// This method acts similarly to using the `Uri` constructor with
   /// some of the arguments taken from this `Uri`. Example:
-  /// ```dart
-  /// Uri uri3 = Uri(
-  ///     scheme: "A",
+  /// ```
+  /// final uri3 = Uri(
+  ///     scheme: 'A',
   ///     userInfo: uri1.userInfo,
   ///     host: uri1.host,
   ///     port: uri1.port,
-  ///     path: "D/E/E",
+  ///     path: 'D/E/E',
   ///     query: uri1.query,
-  ///     fragment: "G");
-  /// print(uri3);  // prints "a://b@c:4/D/E/E?f#G"
-  /// print(uri2 == uri3);  // prints true.
+  ///     fragment: 'G');
```

Comment:
Any chance for more clearer example? like data/images

Reply:
Added new example with real-world uri changes..